widget/accordion.go

```diff
@@ -151,13 +158,17 @@ func (r *accordionRenderer) Layout(size fyne.Size) {
 			d.Resize(fyne.NewSize(size.Width, min))
 			y += min
 		}
+
+		if i < len(r.container.Items)-1 {
+			y += theme.Padding()
+		}
```

Comment:
This adds `padding` between items, but line 145 already adds `padding + 1` between items

Reply:
Correct, see above for explanation